Rovers/Rovers.tex

```diff
@@ -0,0 +1,387 @@
+
+\subsection{Deliverables}
+\label{subsec:deliverables}
+The deliverable includes the technical report, rover software, and rover hardware.
+The primary deliverables of this project will be the technical report and the rover itself by the end of the second semester.
+It is also noted that the materials that the team will come across or notes should also be saved in an Google Drive folder for future reference. 
+The team will also present at SPEX design reviews and the weekly checkups at general meetings with the areas the team members are working on that week.
+The project is intended to last 2 semesters with the deliverable of a mock rover and a technical report.   
+
+\subsubsection{Technical Report}
+\label{techreport}
+The technical report of the rover is a design document complete with the hardware, software, and electrical components of the rover. 
+This report is important as it allows for a future team to easily learn from the past team's experience. 
+This Technical Report will be hosted on GitHub. 
+This will also include the results of the rover tests and the teams recommendations based on them. 
+
+\subsubsection{Rover Software}
+\label{roversoftware}
+The rover software has to be written and hosted on GitHub. 
+This will include rover features such as navigation, control, and power regulation. 
+The rover need to be controlled so software must be made to do so. 
+This means the rover electronics must be integrated with a control application. 
+The rover will need to distribute power to the motors in the wheels so software will need to be made to regulate such power. 
+There will also need to be software for the autonomous or partially autonomous features on the rover. 
+
+\subsubsection{Rover Hardware}
+\label{roverhardware}
+The rover will need to be constructed and tested. Because of this a physically constructed rover is needed. This will be necessary for ImagineRIT for cool factor. This includes the rover body, mast, suspension, and wheels. 
+
+\begin{figure}[hb!]
+  \includegraphics[width=\linewidth]{figs/rocker-bogie.png}
+  \caption{Rocker-Bogie mechanism}
+\label{fig:rockerbogie}
+\end{figure}
+
+\subsection{Milestones}
+\label{subsec:milestones}
+The largest milestones for the technical report would be the would be draft and completion. 
+The rover itself would include the various CAD models of the rover, software milestones, electrical system completion, and drive train system. 
+Other milestones include securing funding and completion of the learning period at the beginning of the fall semester.
+The improvements would be developed in tandem with the MVP, however the MVP will always come first. 
+The improvements will attempted only if there is funding and members to spare. 
+The improvements are varied in complexity and some would take more time than others. 
+Each improvement would be on its own timeline with milestones.
+
+\begin{table}[ht!]
+    \caption{Timeline}
```

Comment:
Where in the timeline will additional funding be requested? How mature will your designs need to be to get the funding required to implement them?